examples/server/server.c

```diff
@@ -2946,6 +2947,9 @@ THREAD_RETURN WOLFSSL_THREAD server_test(void* args)
                     /* discard the packet */
                     n = (int)recvfrom(clientfd, (char *)b, sizeof(b), 0,
                         (struct sockaddr *)&client_addr, &client_len);
+
+                    if (n <= 0)
```

Comment:
I think we should only check for a negative value here. -1 is an error, 0 for tcp is the peer closed, but from recvfrom I think it can be 0. At least from the MacOS man page on it.

Reply:
the preceding `recvfrom(..., MSG_PEEK)` returned positive, so it is always proof of system error if the second `recvfrom()` returns non-positive.  I thought about this at the time, came back to it, and only then realized this nuance.